### ISSUE TITLE
`require` annotation keyword causing issues with openzeppelin

### ISSUE THREAD

**Issue by cd1m0** [ConsenSys/scribble#126]
When trying to instrument the below contract inheriting from open zeppelin:

```
// contracts/simple.sol
// SPDX-License-Identifier: MIT
pragma solidity ^0.8.4;

import "@openzeppelin/contracts/token/ERC20/ERC20.sol";

/// #invariant unchecked_sum(_balances) == _totalSupply;
contract Simple is ERC20 {
    /**  @dev Token Constructor
     */     
    constructor() ERC20("Simple", "Simple") payable {
    }

```

Scribble throws the following error:

```
DEBUG:fuzzing-cli:Initializing configuration context
DEBUG:fuzzing-cli:Parsing config at .fuzz.yml
DEBUG:fuzzing-cli:Initializing tool name middleware with 0.8.1
Scribble has encountered an error (code: 1)
=====STDERR=====
@openzeppelin/contracts/token/ERC20/ERC20.sol:140:16 SyntaxError: Expected "!=", "%", "&", "&&", "*", "**", "+", "-", ".", "/", ";", "<", "<<", "<=", "==", "==>", ">", ">=", ">>", "?", "[", "^", "|", "||", or whitespace but "b" found.

In:


     * required by the EIP. See the note at the beginning of {ERC20}.
     *
     * Requirements:
     *
     * - `sender` and `recipient` cannot be the zero address.
     * - `sender` must have a balance of at least `amount`.
     * - the caller must have allowance for ``sender``'s tokens of at least
     * `amount`.
     */

=====STDOUT=====
```

The issue is that the regex pattern inside `findAnnotations` mistakenly considers the ` * required ....` to be the start of an annotation:

```
    const rx =
        /\s*(\*|\/\/\/)\s*#?(if_succeeds|if_updated|if_assigned|invariant|assert|try|require|define\s*[a-zA-Z0-9_]*\s*\([^)]*\)|macro\s*[a-zA-Z0-9_]*\s*\([^)]*\))/g;

    let match = rx.exec(meta.text);
    ...
```

Honestly I think the best solution is to finally disallow the (already deprecated) annotation keywords without a '#' (as tracked by #125).

If we decide to do this will close this as duplicate of that one.

**Comment by blitz-1306** [comment]
There is also `try` that may be considered as regularly used keyword. So, this issue appears to be only a beginning.

**Comment by cd1m0** [comment]
Fixed by #125 so closing as duplicate
